spec/views/transaction_index_spec.rb

```diff
@@ -0,0 +1,29 @@
+require 'rails_helper'
+
+RSpec.feature 'Transactions index page', type: :feature do
+  before(:each) do
+    visit user_session_path
+    @user = User.create(name: 'Roland', email: 'roland@gmail.com', password: '123456', password_confirmation: '123456')
+    @category = Category.create(name: 'Basketball', icon: 'assets/budgetbulga.png', user: @user)
+    @transaction = Transaction.new(name: 'Computer', amount: 23, category: @category, user: @user)
+    within('form') do
+      fill_in 'Email', with: 'roland@gmail.com'
+      fill_in 'Password', with: '123456'
+    end
+
+    click_button 'Log in'
+    visit category_transactions_path(@category.id)
+  end
+
+  it 'should display category name the transaction belongs to' do
+    expect(page).to have_content('Basketball')
+  end
+
+  it 'should display the transaction amount' do
+    expect(page).to have_content('0$')
+  end
+
+  it 'page path to be transaction path' do
+    expect(page).to have_current_path category_transactions_path(@category.id)
```

Comment:
- [ ] Some of your tests failed, Kindly ensure that all your tests pass successfully.
![image](https://user-images.githubusercontent.com/71874425/179370816-88b18d43-1c6f-43da-9d16-d0a116fa4bc2.png)

Reply:
Thanks @RolandM99 for your feedback. Let me look at a way to resolve the issue

Reply:
Well resolved 👍🏽